cypress/helpers/util.ts

```diff
@@ -0,0 +1,130 @@
+import type { MermaidConfig } from '../../packages/mermaid/src/config.type.js';
+
+type CypressConfig = {
+  listUrl?: boolean;
+  listId?: string;
+  name?: string;
+};
+type CypressMermaidConfig = MermaidConfig & CypressConfig;
+
+interface CodeObject {
+  code: string;
+  mermaid: CypressMermaidConfig;
+}
+
+const utf8ToB64 = (str: string): string => {
+  return window.btoa(decodeURIComponent(encodeURIComponent(str)));
+};
+
+const batchId: string = 'mermaid-batch' + new Date().getTime();
```

Comment:
We should use the commit hash or some env variable that Github actions exposes. 
As our E2E tests run in different machines, the batchID will be different for each shard.

Reply:
I wasn't really sure what `batchId` so left it as it is.

I think this looks good:

```suggestion
const batchId: string = 'mermaid-batch' + execSync('git rev-parse HEAD').toString().trim();
```

---

To import `execSync`, you could to the following:
```ts
import { execSync } from 'child_process';
```

---

Sorry but didn't bring my laptop with me so you can update the branch as you wish :)

Reply:
I think we have to use Cypress.env here as it's gonna run in the browser. Have pushed https://github.com/mermaid-js/mermaid/pull/4552/commits/962ff73fc3d6a1813f364359ed3b7aeec176c631